packages/wallets/src/components/ToggleSwitch/ToggleSwitch.scss

```diff
@@ -0,0 +1,59 @@
+.wallets-toggle-switch {
+    position: relative;
+    display: inline-flex;
+    width: 4.4rem;
+    height: 2.4rem;
+
+    @include mobile {
+        width: 3.8rem;
+    }
+
+    input {
+        width: 0;
+        height: 0;
+    }
+
+    &__slider {
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        cursor: pointer;
+        background-color: var(--light-4-disabled-text, #d6d6d6);
+        transition: 0.4s;
+        border-radius: 16px;
```

Comment:
we're using px or should be moved to rem?

Reply:
We're using rem. Will change this accordingly thanks for pointing it out 🙏🏼